### PR TITLE
LEAF-3895 - Automated Scripts Error Handling

### PR DESCRIPTION
**Pull request by shaneodd**
Description

Update existing automated scripts to return status codes on success/failure
code value
message describing nature of the issue
suggestion to fix
Update CRON class to redirect failure codes to an email forwarded to the dev team
Provide each piece of the status message for devs to read
Testing:

regression testing will be needed, since this is a script this may not be as testable for the testing team. Will need to discuss with testing team on how to hand these items off.

Additional Notes:
Mailer was not installed on the master branch I pulled.

### CODE REVIEW

File: libs/php-commons/ErrorNotify.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-namespace App\Leaf;
+namespace Leaf;
```
Comment (jampaul3) [inline libs/php-commons/ErrorNotify.php:3]: I would put this in the app/Leaf folder and leave the namespace as App\Leaf. We should be moving away from html/libs/php-commons

**Review by jampaul3 — CHANGES_REQUESTED** [review]

**Review by Pelentan — CHANGES_REQUESTED** [review]

File: scripts/scheduled-task-commands/automatedEmailReminder.php [hunk 2]
```diff
@@ -1,67 +1,24 @@
...
-$blacklist = [
-    'pre-apr10',
-    'piwik',
-];
-
-$folder_to_check = 'scripts';
-//$folder_to_check = 'LEAF_Nexus';
+// this file will need to be added, Pete's destruction ticket has it already.
+require_once 'globals.php';
+require_once LIB_PATH . '/php-commons/Db.php';
```
[the remark this answers is not in the document]
Comment (shaneodd) [reply scripts/scheduled-task-commands/automatedEmailReminder.php]: Using the new db class

File: libs/php-commons/ErrorNotify.php (hunk 1 above)
Comment (shaneodd) [reply libs/php-commons/ErrorNotify.php]: Moved to the new location, updated refrences.

File: libs/php-commons/ErrorNotify.php [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+class ErrorNotify
+{
+
+    /**
+     * This is the basic function we are going to start with for getting notifications of errors to devs, we will want to move this over
+     * to dynatrace eventually.
+     * @param string $title
+     * @param array $errorsArr 
+     */
+    public function sendNotification(string $title, array $errorsArr){
```
[the remark this answers is not in the document]
Comment (shaneodd) [reply libs/php-commons/ErrorNotify.php]: Updated.

**Review by jampaul3 — APPROVED** [review]

**Review by Pelentan — APPROVED** [review]

**Review by KCN8 — APPROVED** [review]

**Review by aerinkayne — APPROVED** [review]

**Pull request merged**